client_authentication_jwks_strategy.go

```diff
@@ -60,17 +60,17 @@ func (s *DefaultJWKSFetcherStrategy) Resolve(location string, forceRefresh bool)
 	if !ok || forceRefresh {
 		response, err := s.client.Get(location)
 		if err != nil {
-			return nil, errors.WithStack(ErrServerError.WithHintf(`Unable to fetch JSON Web Keys from location "%s".`, location).WithCause(err).WithDebug(err.Error()))
+			return nil, errors.WithStack(ErrServerError.WithHintf("Unable to fetch JSON Web Keys from location '%s' because: %s.", location, err).WithCause(err).WithDebug(err.Error()))
```

Comment:
Same here. `err` could be sensitive.

Reply:
Hm I don't really think that's the case. `net/http` is basically just saying DIAL errors on hostnames and IPs. Since the location is given by the client anyways I don't think there's much that could leak here?

Reply:
But yeah scrap my last comment - you're right. I've addressed this!